qa/src/test/java/io/camunda/migrator/qa/persistence/DropSchemaTest.java

```diff
@@ -55,45 +56,68 @@ void shouldMigrationSchemaBeKeptOnShutdown() throws Exception {
   @Test
   void shouldMigrationSchemaBeDroppedOnShutdown() throws Exception {
```

Comment:
🔧 include prefix in the name
```suggestion
  void shouldMigrationSchemaWithPrefixBeDroppedOnShutdown() throws Exception {
```

Reply:
Done